dbt_semantic_interfaces/protocols/metric.py

```diff
@@ -215,3 +270,9 @@ def metadata(self) -> Optional[Metadata]:  # noqa: D
     def label(self) -> Optional[str]:
         """Returns a string representing a human readable label for the metric."""
         pass
+
+    @property
+    @abstractmethod
+    def conversion_params(self) -> ConversionTypeParams:
```

Comment:
I see this was removed in a subsequent commit and I think that's the right approach, we don't need this here.